crates/daphne-worker-test/wrangler.storage-proxy.toml

```diff
@@ -71,3 +70,7 @@ tag = "v2"
 renamed_classes = [
     { from = "GarbageCollector", to = "TestStateCleaner" },
 ]
+
+[[migrations]]
+tag = "v3"
+deleted_classes = ["HelperStateStore"]
```

Comment:
I don't think we need to document the migration, since it's not actually part of a deployment.